Signal/src/call/CallService.swift

```diff
@@ -74,7 +74,8 @@ enum CallError: Error {
     case assertionError(description: String)
     case disconnected
     case externalError(underlyingError: Error)
-    case timeout(description: String, call: SignalCall)
```

Comment:
It seems like since you're now similarly pegging most all errors to their `call`, that the new logic ensures the previous timeout behavior. 👍